cmd/kubelet/app/server.go

```diff
@@ -500,11 +500,11 @@ func getReservedCPUs(machineInfo *cadvisorapi.MachineInfo, cpus string) (cpuset.
 
 	topo, err := topology.Discover(machineInfo)
 	if err != nil {
-		return emptyCPUSet, fmt.Errorf("Unable to discover CPU topology info: %s", err)
+		return emptyCPUSet, fmt.Errorf("unable to discover CPU topology info: %s", err)
 	}
 	reservedCPUSet, err := cpuset.Parse(cpus)
 	if err != nil {
-		return emptyCPUSet, fmt.Errorf("Unable to parse reserved-cpus list: %s", err)
+		return emptyCPUSet, fmt.Errorf("unable to parse reserved-cpus list: %s", err)
```

Comment:
We should capitalise the first letter of the error message as per the migration guide: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#remove-string-formatting-from-log-message

I'm curious about where we've mentioned that error messages shouldn't be captalised as we need to fix it :)

Reply:
That document is  for klog style, but here may be golang style, and as golang code style suugested here: https://github.com/golang/go/wiki/CodeReviewComments#error-strings, error strings should not be capitalized.